packages/core/src/hooks/useCall.ts

```diff
@@ -76,10 +77,15 @@ export function useCall<T extends TypedContract, MN extends ContractMethodNames<
  */
 export function useCalls(calls: (Call | Falsy)[], queryParams: QueryParams = {}): CallResult<Contract, string>[] {
   const chainId = useChainId({ queryParams })
-  const { isStatic } = queryParams
+  const { refresh } = useConfig()
 
   const rawCalls = useMemo(
-    () => calls.map((call) => (chainId !== undefined ? encodeCallData(call, chainId, isStatic) : undefined)),
+    () =>
+      calls.map((call) =>
+        chainId !== undefined
+          ? encodeCallData(call, chainId, { ...queryParams, refresh: queryParams.refresh ?? refresh })
```

Comment:
Can it be simplified into `{ refresh, ...queryParams }`?

Reply:
Yes, it is leftover after rename refreshPerBlocks to refresh

Reply:
Yes, then queryParams will override.